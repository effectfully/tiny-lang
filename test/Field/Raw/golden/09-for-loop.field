/**
  * Parsing for loops
  */
for i = 0 to 0 do end;
