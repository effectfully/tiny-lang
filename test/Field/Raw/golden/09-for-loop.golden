Program (Statements [EFor (Var "i") 0 0 (Statements [])])